### ISSUE TITLE
Screen reader issues with dynamic popup

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32766]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When arrowing through the dynamic pop up the screen reader speaks the title "Dynamic Pop Up Panel" before a number of items (possibly top of columns or grid, etc).  The problem is that this sounds like a heading and the user will quickly press the arrow again to move on, but it is actually one of the dynamic options prefixed with this additional speech.  one of these is the "Other Dynamic Button" which reads as "Dynamic Pop Up Panel other Dynamic Button".  As well as being very verbose the user will have moved on and not know this button is present.

STEPS:
1.  Navigate to a dynamic using Alt right arrow, then press tab to move on to the dynamic Pop Up.
2. Press any arrow key multiple times to navigate through the full list of items. Notice the additional speech on some items.
3. Now navigate quickly with arrows as if skimming through the list and notice you end up skipping the items with the additional speech.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6 (4.7 beta)

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
Here's what we could do:

- When first navigating to the popup, say "Select dynamic" instead of "Dynamic pop up panel"
- Stop announcing the panel name every time you arrow to a different page (that's why it currently says "Dynamic pop up panel" a bunch of times)
- Give crescendo and diminuendo proper names (right now, they say "Other dynamic, button" when selected)

<img width="613" height="220" alt="Image" src="https://github.com/user-attachments/assets/96d976ca-d326-40a6-9480-b159020db0a0" />
